cms/data/nav.yml

```diff
@@ -36,8 +36,8 @@ entries:
     entries:
       - label: About DOAJ
         route: doaj.about             # ~~->About:WebRoute~~
-      - label: DOAJ at 20
-        route: doaj.at_20             # ~~->At20:WebRoute~~
+#      - label: DOAJ at 20
+#        route: doaj.at_20             # ~~->At20:WebRoute~~
```

Comment:
These changes seem unrelated, and will take a page offline, is that the intention

Reply:
I'm confused because these are changes committed by @Steven-Eardley that have been pulled in from `master`... I re-fetched and pulled and pushed to this PR and this is still here.